tests/flow/test_bruteforce.py

```diff
@@ -211,3 +211,62 @@ def test_bf_multivalue():
 
     assert_allclose(bf_labels, [keys],  rtol=1e-5, atol=0)
     assert_allclose(bf_distances, [dists],  rtol=1e-5, atol=0)
+
+def test_multi_range_query():
+    dim = 100
+    num_labels = 20000
+    per_label = 5
+
+    num_elements = num_labels * per_label
+
+    bfparams = BFParams()
+
+    bfparams.dim = dim
+    bfparams.metric = VecSimMetric_L2
+    bfparams.multi = True
+    bfparams.type = VecSimType_FLOAT32
+
+    bfindex = BFIndex(bfparams)
+
+    np.random.seed(47)
+    data = np.float32(np.random.random((num_labels, per_label, dim)))
+    vectors = []
+    for label, vecs in enumerate(data):
+        for vector in vecs:
+            bfindex.add_vector(vector, label)
+            vectors.append((label, vector))
+
+    query_data = np.float32(np.random.random((1, dim)))
+
+    radius = 13.0
+    recalls = {}
+    # calculate distances of the labels in the index
+    dists = {}
+    for key, vec in vectors:
+        dists[key] = min(spatial.distance.sqeuclidean(query_data, vec), dists.get(key, np.inf))
+
+    dists = list(dists.items())
+    dists = sorted(dists, key=lambda pair: pair[1])
+    keys = [key for key, dist in dists if dist <= radius]
+
+    for epsilon_rt in [0.001, 0.01, 0.1]:
```

Comment:
Epsilon is not relevant for FLAT range queries. Please refer to the BF range flow test

Reply:
missed this line :+1:

Reply:
But also recall can be asserted to be 100%